### PR TITLE
Adds the blast suit conversion kit

### PR DESCRIPTION
**Pull request by SubatomicTeddybear**
Character: Duncan McGhul
https://baystation12.net/forums/threads/blast-miner-suit-helmet-duncan-mcghul-noblecaos.2965/#post-44472

### CODE REVIEW

**Comment by SubatomicTeddybear** [comment]
All checks have failed. Wunderbar.

File: config/custom_items.txt [hunk 1]
```diff
@@ -584,3 +584,13 @@ item_name: souvenir thunderdome coin
 item_icon: coin_trophy
 item_desc: A grey coin with a sentence engraved that says, "Thunderdome Galactic cup 2558"
 }
+ckey: noblecaos
```
Comment (Datraen) [inline config/custom_items.txt:587]: Missing opening curly brace*.

**Comment by SubatomicTeddybear** [comment]
I am 99% sure I fixed that issue...

EDIT: I just fixed it... twice. Why is it not showing up on git?

**Comment by SubatomicTeddybear** [comment]
10/10, I forgot to actually send it to Github. :|

**Comment by SubatomicTeddybear** [comment]
What'd I screw up, I legit don't see anything.

**Comment by SubatomicTeddybear** [comment]
Nevermind

**Pull request merged**